doc/classes/NavigationMeshSourceGeometryData3D.xml

```diff
@@ -14,23 +14,23 @@
 			<param index="0" name="faces" type="PackedVector3Array" />
 			<param index="1" name="xform" type="Transform3D" />
 			<description>
-				Adds an array of vertex positions to the geometry data for navigation mesh baking to form triangulated faces. For each face the array must have three vertex positions in clockwise winding order. Since [NavigationMesh] resource have no transform all vertex positions need to be offset by the node's transform using the [code]xform[/code] parameter.
+				Adds an array of vertex positions to the geometry data for navigation mesh baking to form triangulated faces. For each face the array must have three vertex positions in clockwise winding order. Since [NavigationMesh] resource have no transform all vertex positions need to be offset by the node's transform using [param xform].
```

Comment:
```suggestion
				Adds an array of vertex positions to the geometry data for navigation mesh baking to form triangulated faces. For each face the array must have three vertex positions in clockwise winding order. Since [NavigationMesh] resources have no transform, all vertex positions need to be offset by the node's transform using [param xform].
```

Likewise below.